R/longmodel.R

```diff
@@ -0,0 +1,117 @@
+
+
+
+
+
+
+
+#' Title - TODO
+#'
+#' Description - TODO
+#'
+#' @slot stan TODO
+#' @export
+LongModel <- setClass(
```

Comment:
Should we add a test for the `LongModel` object constructor ?

Reply:
for sure, apologies this isn't ready for review yet :) Will message when it is. Sorry for the confusion